snomed-database-loader/load_release-postgresql.sh

```diff
@@ -41,7 +41,19 @@ if [[ $1 == *uk_sct2mo* ]]; then
 	isMonolith=true
 fi
 
-databaseUri="postgresql://${PS_DB_OWNER_NAME}:${POSTGRES_PASSWORD}@${PS_DB_HOST}:${PS_DB_PORT}/${dbName}"
+stringencoder() {
```

Comment:
Last thing from me would be - `stringecoder` is a little confusing from an outside-in perspective, some questions I initially had:

* Encoded into what form? (Percent encoding)[https://developer.mozilla.org/en-US/docs/Glossary/Percent-encoding]?